system/Email/Email.php

```diff
@@ -2367,7 +2367,10 @@ public function printDebugger($include = ['headers', 'subject', 'body'])
 
 		is_array($include) || $include = [$include]; // @phpstan-ignore-line
```

Comment:
I know it is out-of-scope, but should `InlineIfToExplicitIfRector` also refactor boolean OR and concatenations? Looking at the code it only deals with `BooleanAnd` and `Assign` types.

Reply:
Yes, BooleanOr should be covered, that maybe next improvement 👍

Reply:
implemented in rector dev-master, wait for next release ;)